css/style.css

```diff
@@ -27,19 +27,6 @@ Developer @GA in NYC
      height: 0;
 }
 
-.clear {
-  clear: both;
-}
-
-.alignright {
-  float: right;
-  padding: 0 0 10px 10px; /* note the padding around a right floated image */
-}
-
-.alignleft {
```

Comment:
I like how you got rid of code that wasn't being used. A real skill once projects get really big.